gctscript/vm/autoload_test.go

```diff
@@ -0,0 +1,38 @@
+package vm
+
+import (
+	"path/filepath"
+	"testing"
+)
+
+const scriptName string = "1D01TH0RS3.gct"
+
+var (
+	GCTConfig = &Config{
+		Enabled: true,
```

Comment:
We've got a lot of Verbose checks which drops test coverage, perhaps set it to true and initiliase a basic logger for the gctscript. Might be helpful for future changes in case a test wigs out and we get more helpful output when trying to diagnose the problem (esp on CIs). What do you think?

Reply:
Yeah sounds like a good idea
I will do the same setup as in database packageof BYO logger that will allow use to use normal built in logger in production or a test friendly one for coverage